### ISSUE TITLE
time window slider below TS plot is not intuitive

### ISSUE THREAD

**Issue by poldrack** [covid19-dash/covid-dashboard#49]
it's not clear to a non-expert user what to do with the time window slider. maybe add some text to say what it does? or a popup?

**Comment by glemaitre** [comment]
Agreed on this point. It was not actually intuitive for me that the below graph was to select a date range.
